### DISCUSSION TITLE
Who is using SQLBoiler in production?

### DISCUSSION THREAD

**Discussion by preslavrachev** [aarondl/sqlboiler#1236] [General]
Hi guys,

We are considering using SQLBoiler for a new project, and were wondering if anyone would be kind enough to share what they have built with it. Obviously SQLBoiler being a very mature project, I am sure that a lot of businesses are relying on it daily. However, having even a vague idea what those businesses are and how they are using it, will make our decision a lot easier. I am sure it will help instill some confidence in other people planning to use it in production too.

Thank you!

**Comment by stephenafamo** [comment]
I'm not sure if a lot of the people using SQLBoiler in production are hanging around the GitHub discussion 😅 

I assume most users will only come to GitHub if there is an issue to report or a question to ask about.

**Comment by christopher-kleine** [comment]
Hi @preslavrachev ,

as @stephenafamo mentioned, most devs don't show up here until they have a problem or something.
I'm guilty of this myself.

But as someone who wrote the SQL queries by hand before and using other ORMs, I might be able to give some insights. Even though it's been ages now and most likely not relevant to you anymore.

We're humans. And using the other ORMs mistakes happened. It was easier to execute them, yes. Even with testing this wasn't detected until it was in production for a while.
SQLBoiler on the other hand takes our Schema from our database. This way the models were always up-to-date and errors were easy to find before it went into production.

SQLBoiler provides useful constants that help to prevent mistakes.
Our code grew in size by a large amount. But since we're not required to change those files by hand, it saved us time and - like I said - prevented many bugs that would have happened during the time of development.

Currently, I'm not able to tell you more about the project, but I absolutely recommend SQLBoiler anytime of the day.
